mail-filter/afew/afew-2.0.0.ebuild

```diff
@@ -0,0 +1,50 @@
+# Copyright 1999-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+DISTUTILS_SINGLE_IMPL=1
+PYTHON_COMPAT=( python{3_6,3_7} )
+
+inherit distutils-r1
+
+DESCRIPTION="Initial tagging script for Notmuch"
+HOMEPAGE="https://github.com/afewmail/afew"
+SRC_URI="mirror://pypi/${PN:0:1}/${PN}/${P}.tar.gz"
+
+LICENSE="ISC"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+IUSE="doc"
+
+RDEPEND="
+	$(python_gen_cond_dep '
+		dev-python/chardet[${PYTHON_MULTI_USEDEP}]
+		dev-python/dkimpy[${PYTHON_MULTI_USEDEP}]
+		net-mail/notmuch[python,${PYTHON_MULTI_USEDEP}]
+	')"
+DEPEND="
+	$(python_gen_cond_dep '
+		doc? ( dev-python/sphinx[${PYTHON_MULTI_USEDEP}] )
```

Comment:
Why is sphinx conditional dependency while in python_compile_all you're using build_sphinx unconditional anyway?

Reply:
Aggreed I removed it.